README.md

```diff
@@ -34,6 +34,47 @@ To use `util.promisify` and `util.callbackify`, Promises must already be availab
 
 See the [Node.js util docs][util].  `util` currently supports the Node 8 LTS API. However, some of the methods are outdated. The `inspect` and `format` methods included in this module are a lot more simple and barebones than the ones in Node.js.
 
+The below table lists the parity of each API with the Node.js version. At the time of writing, the most recent Node.js release is 12.1.0. Any entries with that version are considered up to date; the others may be missing features or bugfixes. See the "History" sections for each API in the [Node.js util docs][util] to learn about the changes that were not yet ported to this module.
+
+> A version marked "<1.2.3" means that changes introduced in Node.js 1.2.3 were _not_ yet ported to this module.
+
+| API | Matching Version |
+|-|-|
+| `callbackify` | 12.1.0 |
+| `debuglog` | 0.11.3 |
+| `format` | <8.4.0 |
```

Comment:
Mentioning the exact version that is reflected seems better than having this notation. There might be more patch releases that could change some small things from the last time this was checked.

Reply:
good point!